include/picongpu/Perf.cpp

```diff
@@ -0,0 +1,57 @@
+// perf.cc
```

Comment:
I already mentioned that this file should not be part of `include/picongpu` because it is required to be pmacc. The problem s that pmacc is that pmacc is still old CMake without target, a way to be installed, and is a header only library. I do not see that we will modernize CMake soon to solve this issue.

Therefore please convert the perf part into s singleton class, move the event map, and all variables from the namespace Performance into this singleton.
This class should be part of pmacc and shipped as `*.hpp` file..

Reply:
Perf.cc and Perf.hpp have now been combined into PerfInfo.hpp, holding a singleton accessible from Environment::PerfInfo()